packages/wallets/src/features/cfd/components/MT5PlatformsList/MT5PlatformsList.tsx

```diff
@@ -29,7 +29,7 @@ const MT5PlatformsList: React.FC = () => {
                 </div>
             </section>
             {!isFetchedAfterMount && <TradingAppCardLoader />}
-            <div className='wallets-mt5-list__content'>
+            <div className='wallets-mt5-list__content' ref={ref}>
```

Comment:
question: if I understand correctly, the case is, that we do want to wait for this specific DIV to be present, before conducting some joyride actions, right? 

so insteadof using mutationobserver (which is somewhat close to "observer" pattern from old stores/mobx, and also its a bit of a sledgehammer), could we pass extra callback to this component, e.g. "onMT5PlatformListLoaded", and then trigger that callback to notify parent component about this list being rendered?

``` 
useEffect(() => {
onMT5PlatformListLoaded();
}, []);
```

So basically insteadof passing ref and then observing dom changes, pass callback and call it when component is rendered with useEffect();
This way, javascript is the source of information needed to render DOM element, and not the other way around. MutationObserver should be a last ditch effort if all standard "react ways" fails

Reply:
not to mention potential performance implication of mutation observer - it does observe all the changes in subtree, so can easily send large number of callbacks really

Reply:
@wojciech-deriv Thanks for your opinion, really appreciate it. I'll try to refactor it and make this with callbacks